packages/unity-js-bridge/src/thirdweb-bridge.ts

```diff
@@ -130,7 +142,7 @@ class ThirdwebBridge implements TWBridge {
     }
     const walletInstance = this.walletMap.get(wallet);
     if (walletInstance) {
-      await walletInstance.connect({ chainId });
+      await walletInstance.connect(chainId);
```

Comment:
why the change here? I switched it to be an object coz there will be more options for other wallets

Reply:
It breaks the types & doesn't build for me